AMD-MAINTENANCE-GUIDE.md

```diff
@@ -0,0 +1,338 @@
+
+This guide details the suggested logic contributors should follow to improve Awesome-MakerDAO.
+
+- Follow the provided guidelines to help preserve a consistent and clean formatting style.
+- Keep Awesome-MakerDAO relevant. 
+  - Add, delete, and relocate resources when appropriate.
+  - Request admin approval before removing resources or when restructuring sections.
+
+**Bounties and Incentives**
+
+The MakerDAO Community Grants program supports members that provide useful contributions and improve Maker's document base. Inquire about bounties and incentives in the #Community-Development channel on MakerDAO's [Rocket.Chat](https://chat.makerdao.com).
+
+**AMD Documents:**
+
+- [Live](https://awesome.makerdao.com/)
+
+## AMD Maintenance Standards
+
+### General
+
+New users should see Awesome-MakerDAO as their starting-point to learn about Maker. Members should be able to share AMD with friends and acquaintances that are looking for a crash course on Maker and its basics.
+
+- Documentation should prioritize:
+    - Informative and comprehensive resources.
+    - A clear and organized structure.
+    - Easy navigation.
+    - Visual appeal.
+- Find and update:
+    - Deviations from the [Writing Style Guide](https://github.com/makerdao/community/blob/master/contributing/style-guide.md).
+    - Dead or outdated links.
+    - Improperly dated resources.
+    - Resources to credit authors.
+
+### Including and Discarding Resources
+
+- Update categories with current and relevant resources.
+  - Add new resources to an appropriate section.
+  - When adding new content, consider the project's age, user base, and whether its main net has been live for at least six months.
+- Discard content that is no longer relevant, repetitive, of low quality, or does not add value.
+- Move outdated content to the [Archive](#archived-resources).
+   - **Do not** move content that is still relevant to a section.
+  - Only move content if it is still interesting, otherwise, discard it.
+   - Some resources may already be eligible for the Archive, especially those over a year old.
+
+### Style and Formatting
+
+#### Writing Style Guide
+
+The MakerDAO Community [Writing Style Guide](https://github.com/makerdao/community/blob/master/contributing/style-guide.md) summarizes the standards and best practices writers should follow when contributing to Awesome-MakerDAO.
+
+- Errors are sometimes overlooked. If something does not adhere to the Writing Style Guide, change it to match.
+- Before starting work on AMD, contributors are encouraged to study the Writing Style Guide as well as the other community contributor guides.
+
+#### Citing and Organizing Content
+
+**Citing**
```

Comment:
best not to mix and match headers `####` and bolding with titles. 

replace with:

```markdown
#### Citing and Organizing Content

#### Citing
```

The same with organizing below.

Reply:
mm... this may need to go into Style guide (or some guide) too then

Reply:
made the change